test/io/db_config.sql

```diff
@@ -23,6 +23,7 @@ ALTER ROLE db_config_authenticator SET pgrst.openapi_server_proxy_uri = 'https:/
 ALTER ROLE db_config_authenticator SET pgrst.server_cors_allowed_origins = 'http://origin.com';
 ALTER ROLE db_config_authenticator SET pgrst.server_timing_enabled = 'false';
 ALTER ROLE db_config_authenticator SET pgrst.server_trace_header = 'CF-Ray';
+ALTER ROLE db_config_authenticator SET pgrst.db_hoisted_tx_settings = 'work_mem';
```

Comment:
This should be the last one before merging. The comment mentions:

> -- these settings will override the values in configs/no-defaults.config, so they must be different

So this should be different from the value on `no-defaults.config` to test that it's actually changing.

Reply:
Ok done.

Btw this whole workflow is double plus ungood, adding a config option is always painful.

This and https://github.com/PostgREST/postgrest/pull/3481#issuecomment-2123572322 tells us the whole Config handling needs reworking.